docs/migrate-to-vector-tiles-v1.md

```diff
@@ -0,0 +1,246 @@
+
+To use vector tiles 1.0, replace your URL with the following scheme, depending on which tile format you want:
+
+- GeoJSON: `http://tile.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.json`
+- TopoJSON: `http://tile.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.topojson`
+- Mapbox Vector Tile: `http://tile.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.mvt`
+
+You need to append your own [API key](https://mapzen.com/documentation/overview/) at the end for access to the full rate limits.
+
+Mapzen offers several different types of tiles in vector and raster formats and the service combines data from multiple sources. Because of this, it was not possible to redirect existing queries automatically to use the 1.0 URL.
+
+## Choose whether to use some or all layers
+
+A vector tiles request can be for one or more individual layers, such as `roads` or `landuse`, or for `all` layers in the service.
+
+With the 1.0 release, you may find benefits to requesting `all` layers instead of custom ones. Specifically, the `all` layers option generally loads faster because these tiles are often served directly from Mapzen's global tile cache. Custom layers experience a small lag because they are extracted, on demand, from `all` tiles. However, custom layer tiles generally have a smaller file size than `all` layers.
+
+You should run benchmark tests for both custom and `all` layers on your target devices and network connections. If you find that one method is clearly faster than the other or better suited to your bandwidth requirements, then choose it. If the results are about the same, then use `all` because it has a simpler service architecture.
+
+If you choose `all`, you can use client-side cartography to limit the display of the layers. For example, if you are using Tangram to draw your map, you can use data [filters](https://mapzen.com/documentation/tangram/Filters-Overview/).
+
+## If you continue using vector.mapzen.com
+
+Although Mapzen announced that the deprecated service will be maintained until October 31, 2017, you should update sooner for the best experience.
+
+During this transition period, only tiles that were previously archived on Mapzen's servers will be available. This means that you may encounter locations, such as rural areas or certain zoom levels, where no data is returned. This can result in a 404 HTTP status error code.
+
+At the time of the 1.0 release, only the `all` and `buildings` layers were archived and cached from the deprecated service. If you attempted to query for other layers, you may have been unable to access them and received an error. This was happening because tiles for custom layers were generated only on demand, and no fresh tiles were being served from the previous URLs. However, shortly after the 1.0 release, Mapzen added special handing for custom layers served from the archive, which gives you the ability to query for any layer on the previous URLs.
```

Comment:
`which restores the ability to query for any layer on the previous URLs.`
from
`which gives you the ability to query for any layer on the previous URLs.`